pallets/automation-time/src/lib.rs

```diff
@@ -179,12 +185,16 @@ pub mod pallet {
 
 	#[pallet::storage]
 	#[pallet::getter(fn get_last_slot)]
-	pub type LastTimeSlot<T: Config> = StorageValue<_, UnixTime>;
+	pub type LastTimeSlot<T: Config> = StorageValue<_, (UnixTime, UnixTime)>;
 
 	#[pallet::storage]
 	#[pallet::getter(fn is_shutdown)]
 	pub type Shutdown<T: Config> = StorageValue<_, bool, ValueQuery>;
 
+	#[pallet::storage]
+	#[pallet::getter(fn get_automation_time_storage_version)]
+	pub type AutomationTimeStorageVersion<T: Config> = StorageValue<_, u16>;
```

Comment:
idk how kosher this is? i feel like they should already have something for storage version, but i can't figure out where to find it